src/analysis/inner_product_space/pi_L2.lean

```diff
@@ -113,6 +113,10 @@ instance : inner_product_space 𝕜 (euclidean_space 𝕜 ι) := by apply_instan
 lemma finrank_euclidean_space_fin {n : ℕ} :
   finite_dimensional.finrank 𝕜 (euclidean_space 𝕜 (fin n)) = n := by simp
 
+lemma euclidean_space.inner_eq_star_dot_product (x y : euclidean_space 𝕜 ι) :
+  ⟪x, y⟫ = matrix.dot_product (star x) y :=
```

Comment:
I'd argue this should be spelt
```suggestion
lemma euclidean_space.inner_eq_star_dot_product (x y : euclidean_space 𝕜 ι) :
  ⟪x, y⟫ = matrix.dot_product (star $ pi_Lp.equiv _ _ x) (pi_Lp.equiv _ _ y) := rfl
```
since that avoids unfolding type synonyms in surprising ways.

I think @hrmacbeth will probably disagree with me though.

Reply:
I don't have an opinion on this, but I've implemented your suggestion.